docs/guidelines/releases-guidelines/07-api-gateway-release-process.md

```diff
@@ -0,0 +1,72 @@
+---
+title: API Gateway release process
+label: internal
+---
+
+This document describes how to create a API Gateway release.
```

Comment:
I think it should be "a**n** API Gateway release.". This is also used in Release content section.